### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by yanghaichao12**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:
Fix a typos

Which issue(s) this PR fixes (optional, in fixes #(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged):
Fixes #

Special notes for your reviewer:

Does this PR introduce a user-facing change?:

NONE

### CODE REVIEW

File: pkg/scheduler/core/extender.go [hunk 1]
```diff
@@ -125,8 +125,8 @@ func (h *HTTPExtender) IsIgnorable() bool {
 	return h.ignorable
 }
 
-// SupportsPreemption returns if a extender support preemption.
-// A extender should have preempt verb defined and enabled its own node cache.
+// SupportsPreemption returns if an extender support preemption.
```
Comment (bsalamat) [inline pkg/scheduler/core/extender.go]: s/returns/returns true/

**Review by bsalamat — COMMENTED** [review]
/ok-to-test

**Review by Huang-Wei — COMMENTED** [review]
@yanghaichao12 please address Bobby's comment. It's almost ready to be merged.

File: pkg/scheduler/core/extender.go (hunk 1 above)
Comment (yanghaichao12) [reply pkg/scheduler/core/extender.go]: yes，it should be “returns true”

**Comment by yanghaichao12** [comment]
@Huang-Wei @bsalamat sorry，last week i was in holiday：） i'll modify it now.

**Comment by yanghaichao12** [comment]
@Huang-Wei @bsalamat  it has been replaced

File: pkg/scheduler/core/extender.go [hunk 2]
```diff
@@ -125,8 +125,8 @@ func (h *HTTPExtender) IsIgnorable() bool {
 	return h.ignorable
 }
 
-// SupportsPreemption returns if a extender support preemption.
-// A extender should have preempt verb defined and enabled its own node cache.
+// SupportsPreemption returns true if an extender support preemption.
```
Comment (bsalamat) [inline pkg/scheduler/core/extender.go]: there is one more grammatical error in this comment:
s/support/supports/

Comment (yanghaichao12) [reply pkg/scheduler/core/extender.go]: i‘ve replaced it

**Comment by yanghaichao12** [comment]
@Huang-Wei @bsalamat hi,is there anything else that need to be modified? :)

**Pull request merged**